forge-game/src/main/java/forge/game/phase/PhaseHandler.java

```diff
@@ -1281,7 +1311,7 @@ public String debugPrintState(boolean hasPriority) {
 
     // just to avoid exposing variable to outer classes
     public void onStackResolved() {
-        givePriorityToPlayer = true;
+//        givePriorityToPlayer = true;
```

Comment:
was it already obsolete or why this change?
Seems to satisfy 117.3b

Reply:
@Agetian It works fine here, Don't really know if this covers 117.3b  correctly. 
![image](https://github.com/Card-Forge/forge/assets/9781539/b76aadf5-d787-4e91-a527-34f06f9029c0)

Reply:
Well my question was did it have to be commented out now to make this fix work at all?